libr/core/cmd_print.c

```diff
@@ -5330,7 +5330,18 @@ static int cmd_print(void *data, const char *input) {
 				goto beach;
 			}
 		}
-		
+
+		if (input[1] == 'x') { // pdx
+			if (input[2] == '?') {
+				r_cons_printf ("Usage: pdx [hexpairs] - disassemble hexpairs\n");
+				return 0;
+			} else {
+				r_core_cmdf (core, "pd @x:%s", r_str_trim_ro (input + 2));
```

Comment:
this can result in command injection issues, will be better to use the API to do that.

Reply:
Okay so I should call `__cmd_pad` function that is correctly implemented instead of the `r_core_cmdf` function before the block computation is resolved for the `pd` command, right? 

What about the `tpx` command? That also does a similar string formatting, wouldn't it be affected by the same command injection?